### PR TITLE
fix: changed number of slides, changed offset between slides, styled card

### PR DESCRIPTION
**Pull request by kate-prakofyeva**
**Description**
Changed the number of slides, changed offset between slides, styled the card

**Type of change**
- [ ]  Feature (introduces new functionality)
- [ ]  Bugfix
- [ ]  Configuration
- [ ]  Refactoring (doesn't change functionality, only the code)
- [x]  Style changes (doesn't affect functionality, only the look of the components)

**PR Checklist**
- [x]  Synced up with latest from the main branch/Code is linted
- [x]  Description of the changes has been added (Screenshots/Video)
- [x]  Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [ ]  Console has been checked for warnings
```
app-index.js:35 Warning: Extra attributes from the server: data-new-gr-c-s-check-loaded,data-gr-ext-installed
at body
at HTML
at InnerLayoutRouter...
```
**Before:**

image
image

**After:**

image
image

### CODE REVIEW

**Review by Timi2424 — APPROVED** [review]
👍

File: src/app/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -47,25 +51,23 @@ export const Slider: React.FC<SliderProps> = ({
...
                 ? Array.from({ length: 8 }).map((_, index) => (
                     <SwiperSlide key={index} className="swiper__slide">
-                      <PartnersCard />
+                      <PartnersCard offset={offset}/>
                     </SwiperSlide>
                   ))
                 : Array.from({ length: 12 }).map((_, index) => (
                     <SwiperSlide key={index} className="swiper__slide">
-                      <StoriesCard />
+                      <StoriesCard offset={offset}/>
```
Comment (SemperAvanti) [inline src/app/components/Slider/Slider.tsx:70]: Czemu to tutaj sa dwa komponenty "ParnersCard" i StoriesCard? czy one nie robia tego samego i nie powinny byc jednym komponentem?

File: src/app/components/SliderComponent/SliderComponent.tsx [hunk 2]
```diff
@@ -19,83 +21,6 @@ export const SliderComponent: React.FC<SliderComponentProps> = ({
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const [swiper, setSwiper] = useState<any | null>(null);
```
Comment (SemperAvanti) [inline src/app/components/SliderComponent/SliderComponent.tsx:22]: Musi byc any?

**Review by SemperAvanti — CHANGES_REQUESTED** [review]

File: src/app/components/Slider/Slider.tsx (hunk 1 above)
Comment (kate-prakofyeva) [reply src/app/components/Slider/Slider.tsx:70]: Dla tego, ze to 2 roznych komponenta :)
StoriesCard:
image
ParnersCard:
image

File: src/app/components/SliderComponent/SliderComponent.tsx (hunk 2 above)
Comment (kate-prakofyeva) [reply src/app/components/SliderComponent/SliderComponent.tsx:22]: Nie dotykalam logiki w tym komponencie, ok, ja to wyprawie

Comment (Timi2424) [reply src/app/components/SliderComponent/SliderComponent.tsx:22]: zajmuję się tym :)

**Pull request merged**